README.md

```
@@ -13,8 +13,11 @@
 > [!NOTE]
 > **Active development is for Windows only.** Linux and macOS builds are untested but available. Bug reports for other platforms are welcome!
 
+## 📚 Documentation
 ## 📚 Documentation
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)